components/03-modules/listitem/_listitem.scss

```diff
@@ -0,0 +1,103 @@
+.products-list-item {
```

Comment:
Use the same name everywhere, so it should be component named `products-list-item` and never write "thingstohertherjustlikethat" as a name 😉